core/src/main/kotlin/org/opensearch/alerting/core/settings/ScheduledJobSettings.kt

```diff
@@ -37,32 +36,38 @@ class ScheduledJobSettings {
 
     companion object {
         val SWEEPER_ENABLED = Setting.boolSetting(
-                "opendistro.scheduled_jobs.enabled",
-                true,
-                Setting.Property.NodeScope, Setting.Property.Dynamic)
+            "plugins.scheduled_jobs.enabled",
+            LegacyOpenDistroScheduledJobSettings.SWEEPER_ENABLED,
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
         val REQUEST_TIMEOUT = Setting.positiveTimeSetting(
-                "opendistro.scheduled_jobs.request_timeout",
-                TimeValue.timeValueSeconds(10),
-                Setting.Property.NodeScope, Setting.Property.Dynamic)
+            "plugins.scheduled_jobs.request_timeout",
+            LegacyOpenDistroScheduledJobSettings.REQUEST_TIMEOUT,
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
 
         val SWEEP_BACKOFF_MILLIS = Setting.positiveTimeSetting(
-                "opendistro.scheduled_jobs.sweeper.backoff_millis",
-                TimeValue.timeValueMillis(50),
-                Setting.Property.NodeScope, Setting.Property.Dynamic)
+            "plugins.scheduled_jobs.sweeper.backoff_millis",
+            LegacyOpenDistroScheduledJobSettings.SWEEP_BACKOFF_MILLIS,
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
 
         val SWEEP_BACKOFF_RETRY_COUNT = Setting.intSetting(
-                "opendistro.scheduled_jobs.retry_count",
-                3,
-                Setting.Property.NodeScope, Setting.Property.Dynamic)
+            "plugins.scheduled_jobs.retry_count",
```

Comment:
I know this is not new, but these names scare me. We have jobscheduler.retry_count in https://github.com/opensearch-project/job-scheduler/blob/1eb8e28823c20de3d3840f3fe755f21a11c5e301/src/main/java/org/opensearch/jobscheduler/JobSchedulerSettings.java#L44 which sounds awfully similar.

I wonder whether this is the time we change it to something more specific to alerting? By adding `alerting` into the name for example?

Reply:
After the chasing the history, the origin design goal is to have same settings to all Job Scheduler based plugins. The central idea seems to be one set of settings to control all jobs run by various plugins. Currently, Alerting, ISM, AD use the same settings. Second part is, when scheduling code is seperated from Alerting to a seperate plugin (Job Schedular) the duplicate code is not removed from alerting. Created https://github.com/opensearch-project/alerting/issues/19 to track this post GA.

Reply:
@dblock, I am fine to keep status-quo for this based on my above analysis.

Reply:
But aside of duplicate code the actual setting names are different:

`jobscheduler.retry_count`
`scheduled_jobs.retry_count`

This wasn't the original intent. When did we break it and what do we do about it?

Reply:
Good callout 👍 . Setting names are different, this is not broken (compared with odfe versions). My 2C is keep status-quo in view of ODFE.1.13-OpenSearch.1.0 compatability and address the names as part of #19

Reply:
Your call. I was just thinking it might be easier to do this now since we're introducing a new setting that fallback to a legacy one.